pkg/filter/utils.go

```diff
@@ -27,7 +27,14 @@ import (
 
 // isSysSchema returns true if the given schema is a system schema
 func isSysSchema(db string) bool {
-	return tifilter.IsSystemSchema(db)
+	switch db {
+	// schema `tidb_cdc` is used in syncpoint function.
+	// Tables in `tidb_cdc` should not be replicated by cdc.
+	case "tidb_cdc":
```

Comment:
Use a constant var please. And also update the code in syncpoint module.

Reply:
done